packit_service/worker/checker/distgit.py

```diff
@@ -199,3 +199,26 @@ def pre_check(self) -> bool:
             )
 
         return valid
+
+
+class IsUpstreamTagMatchingConfig(Checker, GetSyncReleaseTagMixin):
```

Comment:
Maybe I'm missing something, but shouldn't this be used in `get_checkers()` in the handlers?

Reply:
of course it should :D just added